packages/components/psammead-story-promo/src/index.stories.jsx

```diff
@@ -64,6 +75,7 @@ const InfoComponent = ({
         ) : (
           headlineText
         )}
+        <HiddenText headline={headlineText} type={type} />
```

Comment:
Headline text should only be appearing once now. 

Regarding creating a constant like `linkContents`, I'm open to changing things up, but I think it's unnecessary because from what I can tell in Simorgh it mostly retrieves the link content from `item` and then returns something similar to `<HiddenText />`, but in this case I think `<InfoComponent />` pretty much does the retrieval. 

And since headline text and media type are determined by knobs, and the duration is hardcoded - I think the way I've done it is sufficient